concepts/regular-expressions/about.md

```diff
@@ -0,0 +1,124 @@
+# About 
+
+Package [regexp][package-regexp] offers support for regular expressions in Go.
+
+## Syntax 
+
+The [syntax][regexp-syntax] of the regular expressions accepted is the same general syntax used by Perl, Python, and other languages. 
+
+All characters are UTF-8-encoded code points. 
+Following `utf8.DecodeRune`, each byte of an invalid UTF-8 sequence is treated as if it encoded `utf8.RuneError (U+FFFD)`.
+
+It is convenient to write regular expressions as [raw string literals][raw-string-literals] enclosed by backticks.
+This avoids having to quote the backslashes.
+
+## Type `RegExp`
+
+The package defines a type `Regexp` for compiled regular expressions.
+Function `regexp.Compile` compiles a string into a regular expression.
+The function returns `nil` and an error if compilation failed:
+
+```go
+re, err := regexp.Compile(`(a|b)+`)
+fmt.Println(re, err) // => (a|b)+ <nil>
+re, err = regexp.Compile(`a|b)+`)
+fmt.Println(re, err) // => <nil> error parsing regexp: unexpected ): `a|b)+`
+```
+
+Function `MustCompile` is like `Compile` but panics if the expression cannot be parsed.
+It simplifies safe initialization of global variables holding compiled regular expressions.
+  
+There are 16 methods of `Regexp` that match a regular expression and identify the matched text.
+Their names are matched by this regular expression:
+
+```text
+Find(All)?(String)?(Submatch)?(Index)?
+```
+
+* If `All` is present, the routine matches successive non-overlapping matches of the entire expressions.
+* If `String` is present, the argument is a string; otherwise it is a slice of bytes; return values are adjusted as appropriate. 
+* If `Submatch` is present, the return value is a slice identifying the successive submatches of the expression.
+* If `Index` is present, matches and submatches are identified by byte index pairs within the input string.
```

Comment:
I like this bit a lot!

Reply:
Yes, it is cute describing regexp function syntax using regular expressions :)
(It's taken from the regexp package doc)